src/cascade/executor/epiviz_runner.py

```diff
@@ -230,6 +233,10 @@ def main(args):
     """
     ec = make_execution_context()
     settings = load_settings(ec, args.meid, args.mvid, args.settings_file)
+
+    if settings.model.drill != "drill":
+        raise NotImplementedError("Only 'drill' mode is currently supported")
```

Comment:
Ah, right. Nice.

Reply:
Trying to cinch the validation and checks a little tighter every time I touch something.